docs/.vuepress/sidebar/zh-CN.js

```diff
@@ -317,17 +324,18 @@ module.exports = [
             title: "聚合函数",
             directoryPath: "aggregate-functions/",
             children: [
+              "approx_count_distinct",
```

Comment:
Do we have documentation for these function declarations?

Reply:
it is alias of ndv compute approx count distinct use HyperLogLog